yapapi/runner/__init__.py

```diff
@@ -157,12 +158,19 @@ def __init__(
         self._budget_allocation: Optional[rest.payment.Allocation] = None
 
     async def map(
-        self, worker: Callable[[WorkContext, AsyncIterator["Task"]], AsyncIterator[Work]], data
```

Comment:
The return type of `worker` is wrong here, the resulting `AsyncIterator` should return pairs `Tuple[Task, Work]`